packages/apps-config/src/api/spec/dock.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'AccountId',
+  LookupSource: 'AccountId',
```

Comment:
There is no `RefCount` override, so I'm assuming this is all based off the released Substrate 2.0 (final)?

Reply:
Yes.